spec/spec_helper.rb

```diff
@@ -27,6 +29,7 @@
   default_facts.merge!(YAML.load(File.read(File.expand_path('../default_module_facts.yml', __FILE__)))) if File.exist?(File.expand_path('../default_module_facts.yml', __FILE__))
   c.default_facts = default_facts
   c.hiera_config = 'spec/fixtures/hiera/hiera.yaml'
+  c.mock_with :rspec
```

Comment:
I thought rspec is the default and this isn't needed. However, our template actually supports it \o/
https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/spec/spec_helper.rb.erb#L47

Please configure it like this: https://github.com/voxpupuli/puppet-extlib/blob/master/.sync.yml#L5